.github/workflows/run-merge-hive-tests.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Check out Hive
         uses: actions/checkout@v2
         with:
-          repository: 'ethereum/hive'
+          repository: 'DockBoss/hive'
```

Comment:
Move it to NethermindEth, but even better check out official hive repo and just change that file dynamically by some script!